book/translation.rst

```diff
@@ -374,6 +374,18 @@ For more options, see :ref:`component-translator-message-catalogs`.
 
 .. note::
 
+    The paths at which Symfony will look for message files can be modified by adding
+    your custom resources path in your config.yml file:
```

Comment:
I think we can make this more direct and shorter (shorter = better is quite an important rule in documentations): ``You can add other translation paths with the ``paths`` option in the configuration:``